translate/storage/mozilla_lang.py

```diff
@@ -80,6 +80,7 @@ def parse(self, lines):
 
         if not isinstance(lines, list):
             lines = lines.split(b"\n")
+        unit = None
```

Comment:
Nit: move this three lines above to keep all the variable initialization together.